### PR TITLE
Add docker

### PR DESCRIPTION
**Pull request by bryan-havenly**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -
+set -o nounset                              # Treat unset variables as an error
+
+sleep 10
+#npm install
```
Comment (jrichard0725) [inline entrypoint.sh]: What is this? Lol

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+version: '3.2'
+
+services:
+  # db:
+  #   container_name: mysql_test
+  #   image: mysql:5.6
+  #   environment:
+  #     MYSQL_ROOT_PASSWORD: root
```
Comment (jrichard0725) [inline docker-compose.yml]: Why commented out?

File: server/graphql/resolvers/Room.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  isComplete : (room, args, context) => false,
```
Comment (alexweltman) [inline server/graphql/resolvers/Room.js]: can we add a stub for a mutation to mark a room as complete? Then we can ask them to finish it

Comment (alexweltman) [reply server/graphql/resolvers/Room.js]: and also a query to get rooms and boards off rooms? Or do we want them to write it?

**Pull request merged**